tracer/test/snapshots/AspNetCoreMvc31InferredProxySpansTests.Enabled.__path=__statusCode=200_spanCount=3.verified.txt

```diff
@@ -18,33 +18,6 @@
       version: 1.0.0
     }
   },
-  {
-    TraceId: Id_1,
-    SpanId: Id_4,
-    Name: aws.apigateway,
-    Resource: GET /api/test/?,
-    Service: test.api.com,
-    Type: web,
-    ParentId: Id_5,
```

Comment:
This ParentID should NOT be set, there shouldn't be a parent for this span, and this parent was the proxy scope of the previous test as it wasn't closed.